docs/eventing/getting-started.md

```diff
@@ -19,17 +19,20 @@ Use this guide to learn how to create, send, and verify events in Knative. The s
 
 To complete this guide, you will need the following installed and running:
 
-- A [Kubernetes cluster](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) running v1.15 or higher
+- A [Kubernetes cluster](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) running v1.15 or higher.
 
 - [`kubectl` CLI tool](https://kubernetes.io/docs/reference/kubectl/overview/) within a minor version of your Kubernetes cluster.
 
-- [curl v7.65 or higher](https://curl.haxx.se/download.html)
+- [curl v7.65 or higher](https://curl.haxx.se/download.html).
 
-- Knative Eventing Component
+- Knative Eventing component.
 
-   - Knative Eventing In-memory channel 
+- Knative Eventing in-memory channel.
```

Comment:
Hm, why shouldn't the in-memory channel be nested under the Eventing component? I thought a pre-req for the IMC was having Knative Eventing installed.

Reply:
fixed, thx :-)